src/sep8/ApprovalProvider.ts

```diff
@@ -84,4 +88,42 @@ export class ApprovalProvider {
     }
     return res;
   }
+
+  public async postActionUrl(
+    params: PostActionUrlRequest,
+  ): Promise<PostActionUrlResponse> {
+    if (!params.action_url) {
+      throw new Error("Required field 'action_url' missing!");
+    }
+    if (
+      !Object.keys(params.field_value_map) ||
+      !Object.keys(params.field_value_map).length
```

Comment:
I think we don't need the first check here. `Object.keys(params.field_value_map)` will return `[]` if `params.field_value_map` is `{}`, so it will always be valid.

Reply:
https://github.com/stellar/js-stellar-wallets/pull/204/commits/9aa1d72c1334547ad708d37862d6d8eb4d646510